Source/WebCore/rendering/EventRegion.cpp

```diff
@@ -197,18 +197,24 @@ bool EventRegionContext::shouldConsolidateInteractionRegion(IntRect bounds, Rend
         if (!ancestorBounds.contains(bounds))
             return false;
 
+        constexpr auto maxMargin = 50;
         float marginLeft = bounds.x() - ancestorBounds.x();
         float marginRight = ancestorBounds.maxX() - bounds.maxX();
         float marginTop = bounds.y() - ancestorBounds.y();
         float marginBottom = ancestorBounds.maxY() - bounds.maxY();
-
-        constexpr auto maxMargin = 50;
-
-        bool canConsolidate = marginLeft <= maxMargin
+        bool majorOverlap = marginLeft <= maxMargin
             && marginRight <= maxMargin
             && marginTop <= maxMargin
-            && marginBottom <= maxMargin
-            && !renderer.hasVisibleBoxDecorations();
+            && marginBottom <= maxMargin;
+
+        constexpr auto centeredWiggleRoom = 2;
```

Comment:
Not sure about "wiggle room" in the tree (though I love it personally).

Reply:
it is in the tree! but only in tests :D 
went with `offCenterThreshold`